web/src/test/java/com/graphhopper/http/resources/RouteResourceTest.java

```diff
@@ -113,6 +113,12 @@ public void testBasicPostQuery() {
 
         assertTrue(distance > 9000, "distance wasn't correct:" + distance);
         assertTrue(distance < 9500, "distance wasn't correct:" + distance);
+
+        // we currently just ignore URL parameters
```

Comment:
Yeah.. is this intended?

Reply:
Probably not and probably this should not be part of this issue here :)